server/deckUtils.go

```diff
@@ -82,7 +82,7 @@ func generateShuffledDeck(numPlayers int) []model.Card {
 
 	for cardValue, count := range wildCardCounts {
 		for i := 0; i < count; i++ {
-			deck = append(deck, model.Card{"wild", cardValue})
+			deck = append(deck, model.Card{"black", cardValue})
```

Comment:
I'm not certain if I understand. Shouldn't this be "wild". why would this be changed to "black"?

Reply:
ah... I now understand... as this is the color

Reply:
Yeah that was from a different item. I know the pull request is titled unit tests, but this branch is actual-uno-deck so there is a couple more items in it such as this.

Reply:
That will go along with the create new card guis once its merged. It originally was gonna merge into this branch but it was moved to dev.

Reply:
that makes sense.